src/pathfora.js

```diff
@@ -2894,6 +2967,39 @@
 
     /**
      * @public
+     * @description public method to trigger widgets
+     * with the manualTrigger display condition
+     * @param {array}   widgetsIds (optional)
+     */
+    this.triggerWidgets = function (widgetIds) {
+      var pf = this;
+
+      // no widget ids provided, trigger all ready widgets
+      if (typeof widgetIds === 'undefined') {
+        pf.triggeredWidgets['*'] = true;
```

Comment:
Does this ever get flipped back to `false`? This type of one-time state management makes pathfora especially under-equipped for long-running web pages that will need lifecycle control.